src/applications/vaos/new-appointment/components/DateTimeSelectPage/WaitTimeAlert.jsx

```diff
@@ -37,14 +36,14 @@ function ActionButtons(props) {
     <div className="vads-u-display--flex vads-u-margin-top--2 vads-u-align-items--center">
       {eligibleForRequests && (
         <>
-          <Link to={newAppointmentFlow.requestDateTime.url}>
+          <va-link href={newAppointmentFlow.requestDateTime.url}>
             <button
```

Comment:
fix eslint warning (add role="button")

Reply:
@vbahinwillit Hmmm, when I add `role="button"` to the `button` tag I get a new error:

>The element button has an implicit role of button. Defining this explicitly is redundant and should be avoided.

Thoughts?

Reply:
You can replace `button` with the latest `va-button` web component which automatically adds "type=button"

Reply:
@jenniemc Just noting here, the `VaLink` version of `va-link` must be used because we need the React bindings to take the function that is passed to this component.